src/database.ts

```diff
@@ -15,6 +15,7 @@ export const DatabaseIntegration = makeEnum({
   amazondynamodb: 'amazondynamodb',
   amazons3: 'amazons3',
   databrickslakehouse: 'databrickslakehouse',
+  ibmdb2: 'ibmdb2',
```

Comment:
why do we not use camel case here?

Reply:
Is it question for me? I just followed the pattern here